### DISCUSSION TITLE
Perbedaan Normalisasi dan Standarisasi

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#740] [Q&A] [answered]
Permisi saya ingin bertanya apakah normalisasi dan standarisasi itu 2 hal yang berbeda? mengingat tujuan mereka sama, dan biasanya digunakan pada data numerik ya, nah jika konteksnya kedalam Data Teks itu bagaimana ya apakah sama atau berbeda ya terkait standarisasi dan normalisasi ini

Terima Kasih

**Comment by finesaaa** [comment, marked as answer]
Dear pak @daryraa,
Normalisasi dan standarisasi adalah dua konsep yang sering digunakan dalam pengolahan data, terutama pada data numerik, untuk membantu dalam analisis dan pemodelan. Meskipun mereka memiliki tujuan umum yang serupa, yaitu mengubah distribusi data agar lebih mudah diolah, cara kerja dan dampaknya dapat berbeda.

**Normalisasi dan Standarisasi pada Data Numerik:**
- Normalisasi: Mengubah skala data menjadi rentang antara 0 dan 1 untuk perbandingan yang lebih mudah. `(x - min) / (max - min)`
- Standarisasi: Mengubah distribusi data menjadi rata-rata 0 dan deviasi standar 1 untuk memudahkan analisis. `(x - mean) / std`

**Penerapan pada Data Teks:**
- Normalisasi pada Data Teks: Mengubah teks ke format seragam seperti huruf kecil, menghapus tanda baca, dan mereduksi bentuk kata.
- Standarisasi pada Data Teks: Menghilangkan kata-kata "noise" atau "stop words" yang tidak banyak memberikan informasi, untuk fokus pada kata-kata penting.

✨Pada dasarnya, tujuan normalisasi dan standarisasi tetap sama, yaitu membuat data lebih teratur dan mudah diolah, tetapi cara implementasinya dapat bervariasi tergantung pada jenis data yang Anda tangani, baik itu data numerik atau teks.

**Reply by daryraa** [reply]
Terima kasih
